src/Utilities/ImGui/Editors/ResourceEditor.cpp

```diff
@@ -197,10 +195,9 @@ namespace MxEngine::GUI
             ImGui::Text("empty resource");
         }
         
-        static MxString path;
         if (ImGui::Button("load from file"))
         {
-            path = FileManager::OpenFileDialog(0);
+            MxString path = FileManager::OpenFileDialog("*.png *.jpg *.jpeg *.bmp *.tga *.hdr");
```

Comment:
If you ask me, I'd either abstract it in another function like `OpenFileDialogImage` or make a constant, especially if you use this string multiple times